modules/end-user-portal/src/Pages/Apply/UseCase.cshtml.cs

```diff
@@ -0,0 +1,48 @@
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Core.Config;
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Core.Data.Interfaces;
+using GpConnect.NationalDataSharingPortal.EndUserPortal.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+namespace GpConnect.NationalDataSharingPortal.EndUserPortal.Pages.Apply;
+
+public partial class UseCaseModel : BaseModel
+{
+    private readonly ITempDataProviderService _tempDataProviderService;
+
+    public UseCaseModel(IOptions<ApplicationParameters> applicationParameters, ITempDataProviderService tempDataProviderService) : base(applicationParameters)
+    {
+        _tempDataProviderService = tempDataProviderService;
+    }
+
+    public IActionResult OnGetAsync()
+    {
+        ClearModelState();
```

Comment:
Does this need a check that tempDataProvider.hasItems like the other pages?

Reply:
There were some minor inconsistencies around this which I have now corrected.